README.md

```diff
@@ -1,8 +1,761 @@
+      <li><a href="https://symfony.com/doc/current/frontend/ux.html" target="_blank">Symfony UX</a> for progressive JavaScript applications</li>
+      <li><a href="https://symfony.com/bundles/SymfonyMakerBundle/current/index.html" target="_blank">Maker bundle</a> to let you quickly generate code for your controllers, migrations, and more</li>
+      <li><a href="https://symfony.com/doc/current/reference/configuration/web_profiler.html" target="_blank">Symfony Web Profiler</a> to ease debugging issues locally.</li>
+  </ul>
+       </li>
+       <li>A configuration optimized for <strong>PostgreSQL as a database</strong>.</li>
+       <li>A <strong>Docker compose configuration</strong> for PostgreSQL and a <strong>mail catcher</strong> for local development.</li>
+       <li>An <strong>optimized Platform.sh configuration</strong> for Symfony and all included services.</li>
+       <li>A bootstrapped <strong>database migration system</strong>.</li>
+       <li>An optimized <strong>async configuration</strong> for sending emails and notifications asynchronously with retry and failure features enabled on PostgreSQL.</li>
+   </ul>
+</p>
+
+## Getting started
+
+### Deploy
+
+#### Quickstart
+
+
+The quickest way to deploy this template on Platform.sh is by clicking the button below.
+This will automatically create a new project and initialize the repository for you.
 
 <p align="center">
 <a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/symfonycorp/platformsh-symfony-template-metadata/main/template-metadata-sf6.0-php8.1-webapp.yaml&utm_content=symfonycorp&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
-    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
+    <img src="https://platform.sh/images/deploy/deploy-button-lg-blue.svg" alt="Deploy on Platform.sh" width="175px">
+</a>
+</p>
+<br/>
+
+
+You can also quickly recreate this project locally with the following command:
+
+```bash
+#TODO @FabienPotencier add this repo to Packagist 
+# composer create-project symfonycorp/platformsh-symfony-template:6.x -s dev
+```
+
+
+> **Note:**
+>
+> Platform.sh templates prioritize upstream release versions over our own. Despite this, we update template dependencies on a scheduled basis independent of those upstreams. Because of this, template repos do not contain releases. This may change in the future, but until then the `-s dev` flag is necessary to use `composer create-project`.
+
+#### Other deployment options
+
+For all of the other options below, clone this repository first:
+
+```bash
+git clone https://github.com/symfonycorp/platformsh-symfony-template.git
+```
+
+If you're trying to deploy from GitHub, you can generate a copy of this repository first in your own namespace by clicking the [Use this template](https://github.com/symfonycorp/platformsh-symfony-template/generate) button at the top of this page.
```

Comment:
This is another @fabpot todo - marking this repository as a **Template repository** https://github.com/symfonycorp/platformsh-symfony-template/settings